libbeat/docs/shared-instrumentation.asciidoc

```diff
@@ -6,9 +6,10 @@
 ++++
 
 Libbeat uses the Elastic APM Go Agent to instrument its publishing pipeline.
-Currently, only the Elasticsearch output is instrumented.
 To gain insight into the performance of {beatname_uc}, you can enable this instrumentation and send trace data to the APM Integration.
 
+WARNING: Currently, only the Elasticsearch output is instrumented.
```

Comment:
Because this is not going to break the data ingestion/behaviour of Beats, I'd go with note instead of warning. Looking at the [Filestream documentation](https://www.elastic.co/guide/en/beats/filebeat/current/filebeat-input-filestream.html), warning is used for things that will lead to a broken configuraiton/state.